README.md

```diff
@@ -19,97 +20,173 @@ This WordPress project is intended to provide:
 - Homebrew (1.2.3)
   - MariaDB (10.0 or greater)
   - Wordpress Command Line (1.2.1)
+  - Composer (1.4.2)
 - PHP 7
+- Codeception for Wordpress (Wp-Browser 1.21)
+
+***
 
 ## Installation
 
-  Installation process for the setting up Wordpress on a local environment.
+Installation process for the setting up Wordpress on a local environment.
+
+- **Create the MySQL user and database using the MariaDB monitor**.
+
+```
+mysql -uroot
+CREATE DATABASE chappress_dev;
+GRANT ALL PRIVILEGES ON chappress_dev.* TO "chappress"@"localhost" IDENTIFIED BY "chappress";
+FLUSH PRIVILEGES;
+EXIT
+```
+
+- **Create A Testing Database used for Codeception**.
+
+```
+mysql -uroot
+CREATE DATABASE wpTests;
+GRANT ALL PRIVILEGES ON wpTests.* TO "wpTests"@"localhost" IDENTIFIED BY "wpTests";
+FLUSH PRIVILEGES;
+EXIT
+```
+
+- **Download Chap-Press Git with WordPress** 
+
 
-- Create the MySQL user and database using the MariaDB monitor.
+```
+git clone git@github.com:chapmanu/chap-press.git
+cd chap-press/public
+cp -v wp-config.php{-dist,}
 
-  ```
-  mysql -uroot
-  CREATE DATABASE chappress_dev;
-  GRANT ALL PRIVILEGES ON chappress_dev.* TO "chappress"@"localhost" IDENTIFIED BY "chappress";
-  FLUSH PRIVILEGES;
-  EXIT
-  ```
+# The wp-config file already contains the database information to get started.
+# These are for local development only.
+```
 
-- Download WordPress and run the server.
+- **Install Automated Test Suite**
 
-  ```
-  git clone git@github.com:chapmanu/chap-press.git
-  cd chap-press/public
-  cp -v wp-config.php{-dist,}
+Navigate to project folder root: `/chap-press`
 
-  # The wp-config file already contains the database information to get started.
-  # These are for local development only.
+```
+composer install
+#installs depedency from composer.json 
 
-  php -S localhost:8000
-  ```
+echo "alias codecept=./vendor/bin/codecept" >> ~/.bash_profile  
+#creates an alias
 
-  Server should be running in the terminal displaying PHP version, port location, and document root.
+codecept --version  
+```
 
-  Go to your browser and enter the url port:
-  ```
-  http://localhost:8000
-  ```
+Codeception executed as `codecept` or `./vendor/bin/codecept`
 
-  Finish the final instructions through the WordPress installation. The WordPress admin panel should be displayed and the user should have full access to developing in a local environment.
+- **Run the Server**
+  
+Navigate to the `/public` directory and run `php -S localhost:8000`
```

Comment:
Ok, I’ll check it out tomorrow.

My paradigm for running tests is Rails. Not the interface per se (it doesn't have to use rake). But something as simple and automated.